docs/content/overview/product-telemetry.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Product Telemetry"
+description: "Learn how and why NGINX Ingress Controller collects product telemetry data."
+weight: 500
+toc: true
+---
+
+## Overview
+
+NGINX Ingress Controller collects product telemetry data to allow its developers to understand how it's deployed and configured by users.
+The goal is to use this data to triage development work and prioritize features and functionality that will provide the most value to our users.
+
+## About Product Telemetry
+
+Product telemetry is collected by default.
+Data is collected once every 24 hours, and is sent to a service managed by F5 over HTTPS.
+Personally identifiable information (PII) is **not** collected. 
+
+> [!NOTE]  
+> If you prefer to not have data collected, you can [opt-out](#opt-out) when installing NGINX Ingress Controller.
+
+## Data Collected
+
+Below is a list of current set data points that are collected and reported by the NGINX Ingress Controller:
+- **Project Name** This is the name of the product. In this case `NIC`
+- **Project Version** The version of the NGINX Ingress Controller.
+- **Project Architecture** The architecture of the kubernetes environment. (e.g. amd64, arm64, etc...)
+- **Cluster ID** A unique identifier of the kubernetes cluster that the NGINX Ingress Controller is deployed to.
+- **Cluster Platform** The platform that the kubernetes cluster is operating on. (e.g. eks, aks,  etc...)
```

Comment:
```suggestion
- **Project Architecture** The architecture of the Kubernetes environment. (Such as amd64, arm64, etc...)
- **Cluster ID** A unique identifier of the Kubernetes cluster that NGINX Ingress Controller is deployed to.
- **Cluster Platform** The platform that the Kubernetes cluster is operating on. (Such as eks, aks,  etc...)
```
Kubernetes is a proper noun. We've in the middle of working on a new style guide and have decided we're going to try to avoid Latinisms in how we write documentation.